source/DatePicker.js

```diff
@@ -262,7 +253,7 @@
 					second: valueSeconds,
 					millisecond: valueMilliseconds
 				});
-				this.setValue(this.localeValue.getJSDate());
+				this.setValue(localeValue);
```

Comment:
ilib was not a hard dependency for moonstone before. In fact Gray was pretty adamant that it should not be. Is there some reason it has to become a hard dependency now?